### PR TITLE
refactor function name

### PR DESCRIPTION
**Pull request by akanex115**
# Background
issue https://github.com/ajktown/wordnote/issues/44
We refactored the function names in the PR(https://github.com/ajktown/wordnote/pull/158), so the documentation should also apply to them.

## TODOs
- [x] Modify function names to be the same as those in wordnote.

## Checklist (Developers)
- [x] Make this PR as a draft first
- [x] Title is checked
- [x] Background & TODOs are filled
- [x] Assignee is set
- [x] Labels are set
- [x] Project is created & linked, if multiple PRs are associated to this PR
- [x] Appropriate issue(s) is(are) linked to this PR under `development`
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
Please copy and paste the following into your review, ensuring each checkbox is marked as checked
```
- [ ] Check if there are any other missing TODOs that are not yet listed
- [ ] Review Code
- [ ] Every item on the checklist has been addressed accordingly
- [ ] If `development` is associated to this PR, you must check if every TODOs are handled
```

### CODE REVIEW

**Comment by akanex115** [comment]
Review please. Sorry if there are any unnecessary changes.

File: dev/diagrams/on-dev-token-sign-in-handlers.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
-title useDevTokenSignInHandlers()
+title useDevSignIn()
 
 participant "End User" as user
 
 box "FE" #Lightblue
   participant "Component" as fe
-  participant "Hook: useDevTokenSignInHandlers" as hook
+  participant "Hook: useDevSignIn" as hook
   participant "Hook: UseAuthPrep" as hook2
```
Comment (mlajkim) [inline dev/diagrams/on-dev-token-sign-in-handlers.md:25]: I think you can match lowercase here too
```suggestion
  participant "Hook: useAuthPrep" as hook2
```

File: dev/diagrams/on-sign-out-app.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ box
 activate user
```
Comment (mlajkim) [inline dev/diagrams/on-sign-out-app.md:31]: can you update the TOC too?

Please download
https://marketplace.visualstudio.com/items?itemName=huntertran.auto-markdown-toc

File: dev/diagrams/sign-out-hooks.md [hunk 3]
```diff
@@ -7,7 +7,7 @@
   - [Hooks](#hooks)
     - [useOnSignOutApp](#useonsignoutapp)
     - [useIsAppBooted](#useisappbooted)
-    - [useApiErrorHook](#useapierrorhook)
+    - [useApiError](#useapierror)
```
Comment (mlajkim) [inline dev/diagrams/sign-out-hooks.md:10]: Download the TOC and make it automatically updated
https://marketplace.visualstudio.com/items?itemName=huntertran.auto-markdown-toc

**Review by mlajkim — COMMENTED** [review]
First review done

File: dev/diagrams/on-sign-out-app.md (hunk 2 above)
Comment (akanex115) [reply dev/diagrams/on-sign-out-app.md:31]: Handled in [fb0157c](https://github.com/ajktown/docs/pull/57/commits/fb0157cc9544f2b1a9b923a6f0f71d43bb4b3e5d)

File: dev/diagrams/sign-out-hooks.md (hunk 3 above)
Comment (akanex115) [reply dev/diagrams/sign-out-hooks.md]: Handled in [fb0157c](https://github.com/ajktown/docs/pull/57/commits/fb0157cc9544f2b1a9b923a6f0f71d43bb4b3e5d)
OMG!!! Such a useful tool!

File: dev/diagrams/on-sign-out-app.md (hunk 2 above)
Comment (mlajkim) [reply dev/diagrams/on-sign-out-app.md:31]: checked

File: dev/diagrams/on-dev-token-sign-in-handlers.md (hunk 1 above)
Comment (mlajkim) [reply dev/diagrams/on-dev-token-sign-in-handlers.md]: handled in https://github.com/ajktown/docs/pull/57/commits/bea4bf158977a509a87dcff934a242755a678337

File: dev/diagrams/sign-out-hooks.md (hunk 3 above)
Comment (mlajkim) [reply dev/diagrams/sign-out-hooks.md]: @akanex115 hmm it didnt happen to me but good decision that you manually put in.

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**